node/src/sub_lib/utils.rs

```diff
@@ -203,6 +203,11 @@ pub mod tests {
     use masq_lib::multi_config::CommandLineVcl;
     use masq_lib::test_utils::logging::{init_test_logging, TestLogHandler};
 
+    #[test]
```

Comment:
This is my mistake. I should have never brought this constant into life.

Accordingly, please delete your test and also inline the text of the constant in all affected places.